### PR TITLE
March update

### PR DESCRIPTION
**Pull request by sublibra**
Initial march update - might fill in with more corectl updates.

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *core-website* ready!

Built with commit 3c48682e9aa7a9096e92f410ea62816fa8d74433

https://deploy-preview-492--core-website.netlify.com

File: docs/updates/2019-03.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Welcome to the March 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Engine security enhancement
+
+Because of a security concern and to follow best-practices, the Qlik Associative Engine Docker image is changing its
+user and group from `nobody:nobody` to `1910:1910`. You can read more about using an unique UID/GID here:
+[https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user).
+
+If you are on Mac or Linux and using Docker volume mounts, you may need to change the owner of the volume files. You can
```
Comment (peol) [inline docs/updates/2019-03.md]: Maybe make it bold as we did with https://core.qlik.com/updates/2019-01/#abac-moving-from-experimental-to-stable ?

File: docs/updates/2019-03.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+chown -R 1910:1910 /YOUR-MOUNT
+```
+
+## Introducing hamus.js
+
+We have grown tired of typing the same code every time we create a new react.js solution, so we decided to consolidate
+reusable code into a new utility library called [hamus.js](https://github.com/qlik-oss/hamus.js). The repo utilizes
+the new React hook API and offers a set of hooks which simplifies communication with the Qlik Associative Engine.
+
+The repo comes with an example to get you started. Check it out at [hamus.js](https://github.com/qlik-oss/hamus.js).
```
Comment (peol) [inline docs/updates/2019-03.md]: Perhaps also add something like `Initially it contains a set of hooks to simplify the usage of enigma.js models and layouts, as well as rendering picasso.js charts based on hypercubes. We will add more hooks as we identify reusable code in projects, and from feedback. If you have ideas, feel free to share them as issues in the GitHub repository below.`

File: docs/updates/2019-03.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# March 2019
+
+Welcome to the March 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Engine security enhancement
+
+Because of a security concern and to follow best-practices, the Qlik Associative Engine Docker image is changing its
```
Comment (carlioth) [inline docs/updates/2019-03.md]: Perhaps we should add in what version this is introduced (350) ?

File: docs/updates/2019-03.md [hunk 4]
```diff
@@ -9,18 +9,22 @@ Because of a security concern and to follow best-practices, the Qlik Associative
...
-If you are on Mac or Linux and using Docker volume mounts, you may need to change the owner of the volume files. You can
-do that with the following command:
+**If you are on Mac or Linux and using Docker volume mounts, you may need to change the owner of the volume files. You can
+do that with the following command:**
 
 ```bash
 chown -R 1910:1910 /YOUR-MOUNT
 ```
 
+The change is effective from Qlik Associate Engine version 12.345.0.
```
Comment (wennmo) [inline docs/updates/2019-03.md]: Is the change included in version `12.345.0`? Thought it was `12.350.0`?

File: docs/updates/2019-03.md [hunk 5]
```diff
@@ -34,13 +38,38 @@ To increase usability when loading large applications (or with slow connections)
...
 We're in the last phases of finishing a 1.0.0 version of corectl. Focus of this release is to remove the experimental
 flag and make the command structure stable. Along with necessary consistency changes in the API we are adding a
-couple of requested features like bash completion, support for nested JSON structures and better error messages.
+couple of requested features in the latest 0.5.0 release.
+
+### Auto completion
+
+To simplify entering commands, arguments and flags, we have added support for bash completion in bash and zsh. Apart
+from speeding up when entering commands, it lists available objects for `get` commands (dimension, measure, connection
+and app). This will save one call to corectl to retrieve the IDs of the object.
```
Comment (wennmo) [inline docs/updates/2019-03.md]: `get app` is not a command. Bash completions was however also added for `get object ...`.

File: docs/updates/2019-03.md (hunk 4 above)
Comment (sublibra) [reply docs/updates/2019-03.md]: It should be 350!

**Comment by TRB3269** [comment]
Looks good! 💯

**Pull request merged**